com.github.falkTX.Carla.json

```diff
@@ -0,0 +1,140 @@
+{
+  "app-id": "com.github.falkTX.Carla",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "command": "carla",
+  "rename-desktop-file": "carla.desktop",
+  "rename-icon": "carla",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    /* MIDI */
+    "--device=all",
+    /* For pipewire */
+    "--filesystem=xdg-run/pipewire-0",
+    "--system-talk-name=org.freedesktop.RealtimeKit1",
+    /* Allow loading, saving files from the home directory (portals don’t work yet) */
+    "--filesystem=home",
+    "--env=QT_QPA_PLATFORM=xcb",
+    "--env=TMPDIR=/var/tmp",
+    /* On SuSE it seems to be necessary for sound to work */
+    "--env=ALSA_CONFIG_PATH=",
+    "--env=LD_LIBRARY_PATH=/app/lib:/app/lib/libgig",
+    "--env=LV2_PATH=/app/extensions/Plugins/lv2",
```

Comment:
```suggestion
    "--env=LV2_PATH=$HOME/.lv2:/app/extensions/Plugins/lv2",
```

Reply:
Is there a reason you only suggested this for lv2 and not also vst3?
Already saw this on Ardour: https://github.com/flathub/org.ardour.Ardour/pull/32
Is there something bad about loading vst3 plugins from home?

Reply:
added both lv2 and vst3 plugin paths for home now

Reply:
Following the spec for VST I believe it's not necessary, at least for Ardour (I remember the conversation with Robin about it since VST3_PATH was ignored).

I have to check with Carla though.

Reply:
also loading binaries from out of the sandbox is doomed to fail - even more when developers assume Linux = Ubuntu (from a few years ago) x86_64. But that's not what I was getting at here.

Reply:
> I have to check with Carla though.

I tested it now and turns out that Carla needs this path in env to find plugins in home. However it doesn't like to "$HOME" variable. It doesn't resolve to a full path in Carla.
I have to add
`--env=VST3_PATH=/home/elias/.vst3`
for it to work and if I do:
`--env=VST3_PATH=$HOME/.vst3`
Carla won't find the plugins in my user directory.

Reply:
> also loading binaries from out of the sandbox is doomed to fail

Not exactly. Most devs compile against LTS version of Ubuntu. This is old enough to work in Flatpak, so most plugins work just fine.

Reply:
And then they link against `libcurl-gnutls.so` because why not, everybody run Ubuntu, right?

See https://github.com/flathub/org.freedesktop.LinuxAudio.BaseExtension/issues/7

Reply:
I will leave it at this since it is the best we can do for now.